app/services/reporting/total_user_count_report.rb

```diff
@@ -12,6 +12,7 @@ def total_user_count_report
       [
         ['Metric', 'All Users', 'Verified users', 'Time Range Start', 'Time Range End'],
         ['All-time count', total_user_count, verified_user_count, '-', report_date.to_date],
+        ['All-time fully registered', total_fully_registered, '', '-', report_date.to_date],
```

Comment:
let's put a `-` for IDV because it's  intentionally blank
```suggestion
        ['All-time fully registered', total_fully_registered, '-', '-', report_date.to_date],
```